scripts/shared.js

```diff
@@ -0,0 +1,30 @@
+import ProcessData from "/scripts/ProcessData.js";
+
+const cardContainer = document.querySelector(".card-container");
+
+async function getData() {
+  const processor = new ProcessData();
+  return await processor.fetchFolderData();
+}
+
+async function applyHeaderData() {
+  const folderData = await getData();
+
+  const ownerName = document.querySelector(".owner-name");
+  const imgOwner = document.querySelector(".img-owner");
+  const folderName = document.querySelector(".folder-name");
+
+  ownerName.textContent = "@" + folderData.owner.name;
```

Comment:
```suggestion
  ownerName.textContent = `@${folderData.owner.name}`;
```
큰 차이는 없지만 string + string 보단 template literal을 쓰는게 대세인것 같아요.